src/.vuepress/components/common/codepen-snippet.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <p
+    class="codepen"
+    data-theme-id="39028"
+    data-preview="true"
```

Comment:
A lot of the current examples don't have this.

Again, I'm not sure what it does but I was wondering whether the change was intentional?

Reply:
Yes, the same case as I said about `editable`. Initially I thought `preview` should be always true. But my intention here isn't changing already established behavior, so I updated all files that hadn't `data-preview` putting the prop `:preview="false"`.